### ISSUE TITLE
Style settings for Lua are not saved

### ISSUE THREAD

**Issue by vstakhov** [EmmyLua/IntelliJ-EmmyLua#141]
I cannot save/apply settings for Lua style after IDE update using EmmyLua. I'm using CLIon 2018.1.2. Here is how the settings window look like: https://imgur.com/a/riOH4tF I have changed the default values from 4 spaces to 2 spaces but `Apply` button remains inactive and style settings are not applied. I would appreciate any help about how to resolve this issue as it is very annoying.

**Comment by tangzx** [comment]
That's an known issue #138 . Try this build : [1.2.4-eap-23](https://ci.appveyor.com/project/EmmyLua/intellij-emmylua/build/1.2.4-eap-23/artifacts)

**Comment by vstakhov** [comment]
I can now see the preview of the code but settings changes are still not applied nor saved.

**Comment by vstakhov** [comment]
I have also tried to edit this directly in .idea/codeStyles/Project.xml as following:

~~~xml
   <codeStyleSettings language="Lua">
     <indentOptions>                                                                                                                                                                                              
        <option name="INDENT_SIZE" value="2" />                                                                                                                                                                    
        <option name="CONTINUATION_INDENT_SIZE" value="4" />                                                                                                                                                       
        <option name="TAB_SIZE" value="2" />                                                                                                                                                                       
      </indentOptions> 
   </codeStyleSettings>
~~~

But it is still being reset to 4-4-8 default. Settings are wiped upon CLion restart in that file as well.

**Comment by tangzx** [comment]
Fixed! New version => [1.2.4-eap-24](https://ci.appveyor.com/project/EmmyLua/intellij-emmylua/build/1.2.4-eap-24)

**Comment by vstakhov** [comment]
It works now, thank you very much! And thanks for the great plugin!
